Snowflake.Data/Core/SFError.cs

```diff
@@ -88,6 +88,9 @@ public enum SFError
         [SFErrorAttr(errorCode = 270060)]
         INCONSISTENT_RESULT_ERROR,
 
+        [SFErrorAttr(errorCode = 390127)]

```

Comment:
It is the number that was assign to the error https://docs.snowflake.com/en/user-guide/security-mfa#mfa-error-codes

Reply:
Updated code, added additional errors that could be thrown when the cached MFA token is expired or invalid. Multi-factor authentication (MFA) will try to use the passcode from the connection string if available; otherwise, send a Duo push notification to try to authenticate again; if it fails, the token will be removed.

When one of this errors is thrown using cached mfa token, it should be removed from the credential manager.